server/prisma/migrations/20240527134914_switch_to_uuids/migration.sql

```diff
@@ -0,0 +1,780 @@
+ALTER TABLE "TournamentGold" ADD CONSTRAINT "TournamentGold_pkey" PRIMARY KEY ("id");
+
+-- AlterTable
+ALTER TABLE "TournamentXp" DROP CONSTRAINT IF exists "TournamentXp_pkey";
+ALTER TABLE "TournamentXp" ADD COLUMN "temp_id" INT;
+ALTER TABLE "TournamentXp" ADD COLUMN "temp_bruteId" INT;
+UPDATE "TournamentXp"
+SET "temp_id" = "id",
+    "temp_bruteId" = "bruteId";
+ALTER TABLE "TournamentXp" DROP COLUMN "id";
+ALTER TABLE "TournamentXp" DROP COLUMN "bruteId";
+ALTER TABLE "TournamentXp" ADD COLUMN "id" uuid NOT NULL DEFAULT uuid_generate_v4();
+ALTER TABLE "TournamentXp" ADD COLUMN "bruteId" uuid  NOT NULL;
+ALTER TABLE "TournamentXp" ADD CONSTRAINT "TournamentXp_pkey" PRIMARY KEY ("id");
+
+-- AlterTable
+ALTER TABLE "WorkerJob" DROP CONSTRAINT IF exists "WorkerJob_pkey";
+ALTER TABLE "WorkerJob" ADD COLUMN "temp_id" INT;
+UPDATE "WorkerJob" SET "temp_id" = "id";
+ALTER TABLE "WorkerJob" DROP COLUMN "id";
+ALTER TABLE "WorkerJob" ADD COLUMN "id" uuid NOT NULL DEFAULT uuid_generate_v4();
+ALTER TABLE "WorkerJob" ADD CONSTRAINT "WorkerJob_pkey" PRIMARY KEY ("id");
+
+-- AlterTable
+ALTER TABLE "_BruteReportToUser" ADD COLUMN "temp_A" INT;
+UPDATE "_BruteReportToUser" SET "temp_A" = "A";
+ALTER TABLE "_BruteReportToUser" DROP COLUMN "A";
+ALTER TABLE "_BruteReportToUser" ADD COLUMN "A" uuid NOT NULL ;
+
+-- AlterTable
+
+
+
+ALTER TABLE "_BruteToTitle" ADD COLUMN "temp_A" INT;
+ALTER TABLE "_BruteToTitle" ADD COLUMN "temp_B" INT;
+UPDATE "_BruteToTitle" SET "temp_A" = "A", "temp_B" = "B";
+ALTER TABLE "_BruteToTitle" DROP COLUMN "A";
+ALTER TABLE "_BruteToTitle" ADD COLUMN "A" uuid NOT NULL ;
+ALTER TABLE "_BruteToTitle" DROP COLUMN "B";
+ALTER TABLE "_BruteToTitle" ADD COLUMN "B" uuid NOT NULL;
+
+-- AlterTable
+
+
+ALTER TABLE "_BruteToTournament" ADD COLUMN "temp_A" INT;
+ALTER TABLE "_BruteToTournament" ADD COLUMN "temp_B" INT;
+UPDATE "_BruteToTournament"
+SET "temp_A" = "A",
+    "temp_B" = "B";
+ALTER TABLE "_BruteToTournament" DROP COLUMN "A";
+ALTER TABLE "_BruteToTournament" ADD COLUMN "A" uuid NOT NULL ;
+ALTER TABLE "_BruteToTournament" DROP COLUMN "B";
+ALTER TABLE "_BruteToTournament" ADD COLUMN "B" uuid NOT NULL ;
+
+-- AlterTable
+ALTER TABLE "_Opponents" ADD COLUMN "temp_A" INT;
+
+
+ALTER TABLE "_Opponents" ADD COLUMN "temp_B" INT;
+UPDATE "_Opponents"
+SET "temp_A" = "A",
+    "temp_B" = "B";
+ALTER TABLE "_Opponents" DROP COLUMN "A";
+ALTER TABLE "_Opponents" ADD COLUMN "A" uuid NOT NULL ;
+ALTER TABLE "_Opponents" DROP COLUMN "B";
+ALTER TABLE "_Opponents" ADD COLUMN "B" uuid NOT NULL ;
+
+
+-- UPDATE DATAS
+UPDATE "Achievement" SET "bruteId" = "Brute"."id" FROM "Brute" WHERE "Achievement"."temp_bruteId" = "Brute"."temp_id";
+UPDATE "Brute" b1
+SET "masterId" = (
+    SELECT "id"
+    FROM "Brute" b2
+    WHERE b2."temp_id" = b1."temp_masterId"
+)
+WHERE EXISTS (
+    SELECT 1
+    FROM "Brute" b2
+    WHERE b2."temp_id" = b1."temp_masterId"
+);
```

Comment:
I don't understand the need for a WHERE EXISTS here, nor why you don't simply import the other brute with a FROM.
it should work with simply
```sql
UPDATE "Brute" b1 SET "masterId" = b2."id" FROM "Brute" b2 WHERE b2."temp_id" = b1."temp_masterId";
```
Just like you did below

Reply:
Adding `AND b1.temp_masterId IS NOT NULL` would speed up the query too I think